src/components/data/utils.js

```diff
@@ -178,6 +178,14 @@ const useSelectorDefaultFolderPath = () => {
     );
 };
 
+const isPathInTrash = (path, trash_path) => {
+    if (!path || !trash_path) {
+        return false;
+    } else {
+        return path?.startsWith(trash_path);
+    }
+};
```

Comment:
Minor style suggestion, so feel free to skip it:
```suggestion
const isPathInTrash = (path, trash_path) =>
    (path && trash_path && path.startsWith(trash_path));
```